src/Common/tests/System/PlatformDetection.cs

```diff
@@ -43,6 +43,33 @@ public static partial class PlatformDetection
 
         public static int WindowsVersion => GetWindowsVersion();
 
+        public static bool IsAtLeast462()
+        {
+            if (!IsFullFramework)
+            {
+                return true;
```

Comment:
If we return false on !netfx then the test will not be discovered in other platforms but in netfx as ConditionalFact expects true to be ran.